### PR TITLE
DYN-6279  Bump LibG 230 to 3.0.0.2965

### PR DESCRIPTION
**Pull request by sm6srw**
### Purpose

DYN-6279 - Bounding box enhancements

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [X] This PR contains no files larger than 50 MB

### Release Notes

The coordinate system of a bounding box created by BoundingBox.ByMinimumVolume is now less likely to flip its axes when the input geometry varies.

### CODE REVIEW

**Comment by aparajit-pratap** [comment]
@sm6srw I think we should reference LibG build for ASM 230 in all other projects instead of just DynamoCore, no?

**Review by pinzart90 — APPROVED** [review]

**Comment by mjkkirschner** [comment]
@sm6srw any reason not to merge this?

**Comment by sm6srw** [comment]
> @sm6srw I think we should now start referencing the LibG package for ASM 230 in all other projects instead of just DynamoCore, no?

Yes, I guess we should. I will fix that.

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
     <PackageReference Include="Greg" Version="3.0.0.2955" />
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2855" GeneratePathProperty="true" />
```
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj:31]: Is there a newer version of LibG_229 as well? On second thoughts, how about completely removing it?

**Comment by aparajit-pratap** [comment]
@sm6srw checks seem to have failed due to this test failure: [PackageSearchDialogSearchTestStatusFilters](https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/11626/testReport/Dynamo.PackageManager.Wpf.Tests/PackageManagerSearchElementViewModelTests/PackageSearchDialogSearchTestStatusFilters/)
Can you check?

**Comment by sm6srw** [comment]
> @sm6srw checks seem to have failed due to this test failure: [PackageSearchDialogSearchTestStatusFilters](https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/11626/testReport/Dynamo.PackageManager.Wpf.Tests/PackageManagerSearchElementViewModelTests/PackageSearchDialogSearchTestStatusFilters/) Can you check?

This test also fails for other PRs so it looks like it is in master.

File: src/DynamoCore/DynamoCore.csproj (hunk 1 above)
Comment (sm6srw) [reply src/DynamoCore/DynamoCore.csproj:31]: I don't know if I want to remove it just yet. Many devs a probably relying on 229 still.
I will check if there is a newer 229 version.

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**